tests/suites/teleporter/teleporter_suite_test.go

```diff
@@ -57,7 +58,7 @@ var _ = ginkgo.BeforeSuite(func() {
 	Expect(err).Should(BeNil())
 
 	// Create the local network instance
-	ctx, cancel := context.WithTimeout(context.Background(), 120*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 120*2*time.Second)
```

Comment:
Would this make more sense as `60*4*time.Second`?

Reply:
2 corresponds to the number of subnets that are being created and converted.